docs/howtos/locally-published-components-in-ios.md

```diff
@@ -1,24 +0,0 @@
-# Using locally-published components in Firefox for iOS
```

Comment:
👍 It's on my plate to figure out how to have an easy locally-published flow for iOS given this new setup, but I think this will be very important for us to get our components tested before they are consumed by the app